Doc/source/designspaceLib/scripting.rst

```diff
@@ -195,16 +195,16 @@ this into something clever.
     # you can specify a different location for a glyph
     glyphData['instanceLocation'] = dict(weight=500)
 
-    # You can specify different masters
+    # You can specify different sources
     # for this specific glyph.
-    # You can also give those masters new
+    # You can also give those sources new
     # locations. It's a miniature designspace.
     # Remember the "name" attribute we assigned to the sources?
-    glyphData['masters'] = [
-        dict(font="master.thin",
+    glyphData['sources'] = [
```

Comment:
Here you may need to change the matching code that expects this dictionary key

Reply:
are you suggesting to change the occurrences of `glyphData['masters']` with `glyphData['sources']` in `Lib/fontTools/designspaceLib/*` source files?

Reply:
Yes basically, however I'm not familiar with that part of the designspace spec, maybe @LettError and others can advise. I think maybe font editors who rely on the format would need updating as well if the key changes from "masters" to "sources", that is, code outside this repository would need to adapt. But I'm not sure 100%.